# Traceback
# Cтек вызовов в Python
# Теперь поговорим немного про стек вызовов (traceback)
# Программа на Python состоит из блоков, входящих один в другой, поэтому у любой точки программы есть вложенность
# Давайте посмотрим, какую информацию дает стек вызовов, когда указывает на место ошибки.
# Например, вычисление частного двух чисел, введенных пользователем. Отметим, что огромное количество ошибок происходит из-за того, что пользователь ввел неправильные данные.


def div(a, b):
    return a / b

def run():
    a = float(input('Введите a: '))
    b = float(input('Введите b: '))
    print(div(a, b))

run()